mmdet/models/detectors/panoptic_fpn.py

```diff
@@ -0,0 +1,35 @@
+from ..builder import DETECTORS
+from .panoptic_two_stage_segmentor import TwoStagePanopticSegmentor
+
+
+@DETECTORS.register_module()
+class PanopticFPN(TwoStagePanopticSegmentor):
+    """Implementation of Panoptic FPN.
+
+    https://arxiv.org/pdf/1901.02446.
```

Comment:
See example of other detectors https://github.com/open-mmlab/mmdetection/blob/master/mmdet/models/detectors/cascade_rcnn.py#L7